libs/components/modals/src/lib/modules/modal/modal-header.component.scss

```diff
@@ -5,6 +5,13 @@
 h1 {
   margin: 0;
   line-height: 1.2;
+  display: inline;
+}
+
+::ng-deep {
```

Comment:
You won't need this once you use the `sky-control-help-container` class to wrap the `.sky-control-help` content.